armadactl.md

```diff
@@ -0,0 +1,48 @@
+# Command line tool (armadactl)
+
+Armadactl is a command-line tool used for managing jobs in the Armada workload orchestration system. It provides functionality for creating, updating, and deleting jobs, as well as monitoring job status and resource usage.
```

Comment:
Yeah I agree with Kevin, I think a lot of this is missleading. For example, armadactl does not manage jobs, it just helps you submit, watch and cancel them. I would argue Armada itself is what is managing jobs.

Reply:
So, how should we proceed with this? Because we'll be adding `armadactl` on the website and then we'll have to make changes again

Reply:
I would want to see an attempted re-write. Try and make it clear that it is not managing, but communicating.

https://kubernetes.io/docs/reference/kubectl/ I think this is a good refrence